cf/dimensioncoordinate.py

```diff
@@ -334,6 +334,106 @@ def direction(self):
 
         return _direction
 
+    @classmethod
+    def create_regular(
+        cls, args, units=None, standard_name=None, bounds=True
+    ):
+        """
+        Create a new `DimensionCoordinate` with the given range and regular
+        cellsize.
+
+        .. versionadded:: 3.15.0
+
+        :Note: This method does not set the cyclicity of the
+         `DimensionCoordinate`.
+
+        :Parameters:
+
+            args: `sequence of numbers`
+                A sequence of three numeric values representing the lower
+                and upper bounds of the dimension and the cellsize.
+                The first two values in the sequence represent the range,
+                and the third value represents the cellsize.
```

Comment:
Perhaps reword this to not use the word "bounds", as that could be misleading

"A sequence of three numeric values. The first two values in the sequence represent the coordinate range (see the *bounds* parameter for details),  and the third value represents the cellsize."